storage/bootstrap/bootstrapper/fs/source.go

```diff
@@ -116,11 +116,7 @@ func (s *fileSystemSource) enqueueReaders(
 	readersCh chan<- shardReaders,
 ) {
 	for shard, tr := range shardsTimeRanges {
-		var files []string
-		fs.ForEachInfoFile(s.filePathPrefix, namespace, shard, s.readerBufferSize, func(fname string, _ []byte) {
-			files = append(files, fname)
-		})
-
+		files := fs.ReadInfoFiles(s.filePathPrefix, namespace, shard, s.readerBufferSize)
```

Comment:
All good. I think standardising on where we're reading data from is worth the overhead, so earlier we used blockstart time extracted from filename and from the info file both, in separate code-paths. This way at least we avoid doing that. 

Also, wrt to the overhead, `readInfo` is only doing unmarshalling so I don't think it's it's going to be much of a change.